app/imports/ui/pages/TestContract.jsx

```diff
@@ -13,16 +14,15 @@ class TestContract extends React.Component {
   }
 
   deployButton() {
-    deployedContracts.push(deployContracts(contracts[0]));
+    deployedContracts.push(deployContract(contracts[0]));
   }
 
   callContractButton() {
-    payRentFunction(contracts[0]);
+    payRentScheduler(contracts[0]);
   }
 
-  checkContract() {
-    contractCheck();
-    contracts = [];
+  showTransactions() {
+    showTransactions(contracts[0]);
```

Comment:
any reasons to put this in the console instead of in a list in the UI?

Reply:
should we consider writing transactions to mongo? I could do this as a next task.

Reply:
I figured the transactions can just be stored as an array of objects in the smart contract schema.